src/pdb/mod.rs

```diff
@@ -325,50 +255,62 @@ pub struct Page {
     /// > always 0, except 1 for history pages, num entries for strange pages?"
     #[allow(dead_code)]
     unknown7: u16,
+
+    #[br(temp)]
+    #[bw(ignore)]
+    #[br(calc = if num_rows_large > num_rows_small.into() && num_rows_large != 0x1fff { num_rows_large } else { num_rows_small.into() })]
+    num_rows: u16,
+
+    #[br(temp)]
+    #[bw(ignore)]
+    #[br(calc = if num_rows > 0 { (num_rows - 1) / RowGroup::MAX_ROW_COUNT + 1 } else { 0 })]
```

Comment:
unfortunately this is still nightly-only, but I think we should leave this in the code so we can go back and replace it eventually once its been stabilized.
```suggestion
    #[br(calc = num_rows.div_ceil(RowGroup::MAX_ROW_COUNT))]
```

Reply:
https://github.com/Holzhaus/rekordcrate/pull/64/commits/359db9d84ef7a9b40c1e0299ed19a8dadec11dd0